### PR TITLE
Added the option to add fields and Tables

### PR DESCRIPTION
**Pull request by Bertverbeek4PS**
through the API.

### CODE REVIEW

**Comment by DuttaSoumya** [comment]
Hi @Bertverbeek4PS 

I have created a new pull request making some big changes from your PR,
* I have created new APIs solely for the purposes of insertions.
* I have kept the old table API page largely unchanged, while adding the capability to enable/ disable/ reset the individual records
* I removed all system audit fields except systemID
* I added a validation check to ensure that fields are not inserted for absent tables.

Please see the PR at https://github.com/microsoft/bc2adls/pull/113.

I created the PR as it was easier for me to make the changes instead of making comments on this one. If you wish, you may copy my changes to this PR and I shall happily approve it.

best regards,
Dutta.

**Comment by Bertverbeek4PS** [comment]
Hi @DuttaSoumya I have added new changes to the commit based on your proposal :)

File: businessCentral/src/ADLSEFieldAPI.Page.al [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            repeater(GroupName)
+            {
+                field(tableId; Rec."Table ID") { }
+                field(fieldId; Rec."Field ID") { }
+                field(enabled; Rec.Enabled) { }
+                field(systemId; Rec.SystemId)
+                {
+                    Editable = false;
+                }
+                field(lastModifiedDateTime; Rec.SystemModifiedAt)
```
Comment (DuttaSoumya) [inline businessCentral/src/ADLSEFieldAPI.Page.al:31]: Could we please not include `lastModifiedDateTime`, as the `systemRowVersion` should be sufficient to track changes.

File: businessCentral/src/ADLSETableAPI.Page.al [hunk 2]
```diff
@@ -30,15 +34,56 @@ page 82565 "ADLSE Table API"
                 {
                     Editable = false;
                 }
+                field(systemRowVersion; Rec.SystemRowVersion)
```
Comment (DuttaSoumya) [inline businessCentral/src/ADLSETableAPI.Page.al:37]: Could we please exclude lastModifiedDateTime, as the systemRowVersion should be sufficient to track changes.

**Review by DuttaSoumya — CHANGES_REQUESTED** [review]
I am requesting removal of the lastUpdatedTimeStamp field please

**Comment by Bertverbeek4PS** [comment]
I have also changed the ADLSE Setup API and ADLSE Run API to be consistent

**Review by DuttaSoumya — APPROVED** [review]
Looks super good- thanks @Bertverbeek4PS !

**Pull request merged**